docs/modules/ROOT/pages/includes/kafka-streams-processor-configuration-keys.adoc

```diff
@@ -25,6 +25,36 @@ Environment variable: `+++KAFKASTREAMSPROCESSOR_INPUT_TOPIC+++`
 endif::add-copy-button-to-env-var[]
 -- a| string
 |
+a| [[kafka-streams-processor-configuration-keys_kafkastreamsprocessor.input.topics]]`link:#kafka-streams-processor-configuration-keys_kafkastreamsprocessor.input.topics[kafkastreamsprocessor.input.topics]`
+
+
+[.description]
+--
+The Kafka streams processor property to define multiple topics for incoming messages (comma separated list).
+
+ifdef::add-copy-button-to-env-var[]
+Environment variable: env_var_with_copy_button:+++KAFKASTREAMSPROCESSOR_INPUT_TOPICS+++[]
+endif::add-copy-button-to-env-var[]
+ifndef::add-copy-button-to-env-var[]
+Environment variable: `+++KAFKASTREAMSPROCESSOR_INPUT_TOPICS+++`
+endif::add-copy-button-to-env-var[]
+-- a| string
+|
+
+a| [[kafka-streams-processor-configuration-keys_kafkastreamsprocessor-input-sources-source-topic]]`link:#kafka-streams-processor-configuration-keys_kafkastreamsprocessor-input-sources-source-topic[kafkastreamsprocessor.input.sources.<source>.topic]`
+
+[.description]
+--
+The Kafka topic for incoming messages for the given source name.
+
+ifdef::add-copy-button-to-env-var[]
+Environment variable: env_var_with_copy_button:+++KAFKASTREAMSPROCESSOR_INPUT_SOURCES__source__TOPIC+++[]
```

Comment:
no way to put \_\<source uppercase\>\_ ?

Reply:
@bsoaressimoes I think @edeweerd1A wanted to have `<source uppercase>` litterally, not `SOURCE`.
The idea is to:
1. clearly differentiate the part of the variable forced by the property (full uppercase), from the part that depends on the user's topology (source name), 
2. but also to highlight that the source name must be uppercase in final environment variable

You would have `KAFKASTREAMSPROCESSOR_INPUT_SOURCES_PING_TOPIC=ping-events` for instance